DataTables.Blazor/Interop/DomEventListener.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DataTables.Blazor.Interop
+{
+    /// <summary>
+    /// Controls the events between .NET and JS for DataTables.Blazor.
+    /// </summary>
+    public interface IDomEventListener
+    {
+        /// <summary>
+        /// Add event listener for datatable.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="dom"></param>
+        /// <param name="eventName"></param>
+        /// <param name="callback"></param>
+        /// <returns></returns>
+        Task AddAsync<T>(ElementReference dom, string eventName, Action<T> callback);
+
+        /// <summary>
+        /// Clear all listeners and dispose.
+        /// </summary>
+        void Dispose();
```

Comment:
Probably worth this inheriting from `IDisposable`

Reply:
Done